Formula/libcbor.rb

```diff
@@ -13,9 +13,11 @@ class Libcbor < Formula
 
   depends_on "cmake" => :build
 
+  uses_from_macos "cjson" => :build
+
   def install
     mkdir "build" do
-      system "cmake", "-G", "Unix Makefiles", "..", *std_cmake_args
+      system "cmake", "..", *std_cmake_args
```

Comment:
Let's try this. If someone could confirm that this fixes the problem on Linuxbrew that would be great.
```suggestion
      system "cmake", "..", "-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF", *std_cmake_args
```
(I am---not yet at least---a linuxbrew-core maintainer, so I cannot very easily modify Issy's test PR)

Reply:
@xanderlent it appears you did not check the box to allow maintainers to commit to your branch (or it won't let me do it through the GH web UI)